models/issues/comment.go

```diff
@@ -201,10 +204,16 @@ func stringToRoleDescriptor(role string) RoleDescriptor {
 	switch role {
 	case "Poster":
 		return RoleDescriptorPoster
-	case "Writer":
-		return RoleDescriptorWriter
 	case "Owner":
 		return RoleDescriptorOwner
+	case "Member":
```

Comment:
From a semantic point of view, I think this whole method is completely useless.
I don't know why it would ever be needed in the direction `English string` -> `role` instead of the other way around.

---

Ah, I just looked at the implementation.
As we always query for literals, it makes more sense from a readability perspective.

Reply:
I would prefer to totally drop this `stringToRoleDescriptor`, it's clearer to introduce some function like: `HasRoleOwner` / `HasRoleFirstTimeContributor`.

The line of code won't change too much, while the template code will be more readable: `{{if .ShowRole.HasRoleMember}}`

Reply:
Yes, so that we can have translations for this roles.